pkg/agent/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -326,3 +326,11 @@ func (c *Controller) watchNetworkPolicies() {
 		}
 	}
 }
+
+func (c *Controller) GetNetworkPolicyControllerInfo() map[string]int {
```

Comment:
Feel Controller is too heavy to be injected into Monitor, not easy to unit test and exporting too many interfaces. Maybe define an interface NetworkPolicyInfoExporter or Querier, which defines GetXXXXNum methods and only inject its instance to monitor, so do other modules requiring the data.

Reply:
OK, let me take a look at this part.